src/ListView.vala

```diff
@@ -309,7 +309,14 @@ public class Tasks.ListView : Gtk.Grid {
             task_list.add (task_row);
             return true;
         });
-        task_list.show_all ();
+
+        Idle.add (() => {
+            task_list.invalidate_sort ();
+            task_list.invalidate_headers ();
```

Comment:
Turns out calling `invalidate_sort` already calls `invalidate_headers`, so you don't need to do both :)

Reply:
Good catch!